ChemistryLib/PhreeqcIOData/CreateKineticReactant.h

```diff
@@ -12,18 +12,17 @@
 #include <boost/optional/optional.hpp>
 #include <vector>
 
+#include "KineticReactant.h"
```

Comment:
Why is the change from fwd-decl needed?

Reply:
I think the header file `KineticReactant.h` has to be included in the implementation file `CreateKineticReactant.cpp`, because the ctor of class `KineticReactant` is applied in this file for instance creation. If so, I can move this header file into `CreateKineticReactant.h` and drop the fwd-declaration. If my understanding is not correct, I can add the fwd-decl back and leave the header file in the implementation file.

Reply:
Fixed.